include/dxc/DxilContainer/DxilPipelineStateValidation.h

```diff
@@ -16,6 +16,12 @@
 #include <cstring>
 #include <stdint.h>
 
+#include <assert.h>
```

Comment:
nit: why the line break between this and the other includes?

Reply:
Fixed.